app/assets/scripts/views/category.js

```diff
@@ -127,7 +127,7 @@ var Category = React.createClass({
             </div>
             <div className='inpage__header-data'>
               <ul className='inpage-stats'>
-                <li> {shortTally(totalBudget)} <small>{t.stat_one}</small></li>
+                <li> ${shortTally(totalBudget)} <small>{t.stat_one}</small></li>
```

Comment:
Same as the above comment, let's make another function that accounts for currency in `format` rather than do the formatting in the template for each file.